locales/es/LC_MESSAGES/help.po

```diff
@@ -41,30 +46,30 @@ msgstr ""
 
 #: helpFuncs.py:28
 msgid "Due to the way Telegram creates and limits inline buttons, there can't be displayed more than 23 rows of buttons. Therefore, I restricted the items per list to a maximum of 20 items."
-msgstr ""
+msgstr "Debido a la forma en la que telegram crea y limita los botones, no se pueden mostrar mas de 23 filas de botones. Por ello, se limitan a un maximo de 20 elementos por lista"
 
 #: helpFuncs.py:31
 msgid "If you want to make a new list, use the /new command or the inline function, type in what title you want and that's it."
-msgstr ""
+msgstr "Si quieres crear una nueva lista, usa el comando '/new' o función en linea, escribe el titulo que quieras y ya estará creada."
 
 #: helpFuncs.py:33
 msgid "This bot uses a permission system. Per default, nobody except you can modify a list. To change this, you have to go to the list settings and press the upper left button (👤) and the ones you want to work on the list needs to press the corresponding link of the list you shared. Please remember that as long as your list is marked as open, anyone with the link/code can join and manipulate it, so remember to close it again :)"
-msgstr ""
+msgstr "Este bot usa un sistema de permisos, por defecto nadie puede modificar tus listas. Para cambiar esto, tienes que ir a la configuración de la lista y pulsar el botón de arriba a la izquierda (👤) y después compartir la lista con quien quieras que pueda editarla. Por favor, recuerda que al marcar la lista como abierta, cualquiera con quien compartas la lista tendrá acceso total a la lista, aunque siempre puedes cerrarla de nuevo :)"
 
 #: helpFuncs.py:38
 msgid ""
 "👤 means, that the list is currently closed and usage is restricted to the owner and users, who already joined the list. Pushing this button will open the list for everyone and will change the icon to 👥.\n"
 "✅⬆ and ✅⬇ will reorder already done tasks to the top/bottom of the list, respectively.\n"
 "↩ to go back to the list."
-msgstr ""
+msgstr "👤 significa que la lista esta cerrada y el uso esta restringido a el dueño y los usuarios que ya se añadieron a la lista, Pulsar esta botón abrirá la lista para cualquiera y cambiara el icono a 👥.\n"
 
```

Comment:
May you translate the other two lines too, please? :3

Reply:
wow Sorry ;)